src/transformState.js

```diff
@@ -6,6 +6,23 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
It's better to use here switch case :
`case 'addProperties':

case 'removeProperties':
case 'clear':
`